src/cpu/aarch64/acl_prelu.cpp

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+* Copyright 2022 Arm Ltd. and affiliates
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "cpu/aarch64/acl_prelu.hpp"
+
+namespace dnnl {
+namespace impl {
+namespace cpu {
+namespace aarch64 {
+
+template <data_type_t data_type>
+status_t acl_prelu_fwd_t<data_type>::execute_forward(
+        const exec_ctx_t &ctx) const {
+    // Lock here is needed because resource_mapper does not support
+    // concurrent multithreaded access.
+    std::lock_guard<std::mutex> _lock {this->mtx};
+
+    status_t status = status::success;
+
+    auto src = CTX_IN_MEM(const data_t *, DNNL_ARG_SRC);
```

Comment:
Can it be `void` instead of `data_t` since only the pointer passed? If yes, I'd suggest to get rid of templated argument at all.

Reply:
Sure thing.